### PR TITLE
Fixing problems with text of the concept and exercise.

### PR DESCRIPTION
**Pull request by verdammelt**
The about.md was only an additional note rather than a recapitualation
with additional note. The blurb was for an earlier idea for the
exercise. The introduction talked about character codes but the
exercise no longer used it.

### CODE REVIEW

File: exercises/concept/character-study/.meta/config.json [hunk 1]
```diff
@@ -5,5 +5,5 @@
     "test": ["character-study-test.lisp"],
     "exemplar": [".meta/exemplar.lisp"]
   },
-  "blurb": "Learn the basics of Common Lisp characters by working on cyphers"
+  "blurb": "Learn the basics of Common Lisp characters by helpeing Lewis learn about Earth letters."
```
Comment (TheLostLambda) [inline exercises/concept/character-study/.meta/config.json:8]: It seems that all of Exercism is a little inconsistent about periods at the ends of blubs – most seem to not have a period at the end, but I've also found a couple that do.

It's a small style thing I'll leave up to you!

File: concepts/characters/introduction.md [hunk 2]
```diff
@@ -12,13 +12,6 @@ Some non-graphical characters such as space or new-line have names such as `#\Sp
 The standard only requires an implementation to have 96 characters (upper and lower case Latin alphabetic characters (`A-Za-z`), the digits (`0-9`), space (`#\Space`), newline (#\Newline) and punctuation (e.g. ``!$"'(),_-./:;?+<=>#%&*@[\]{|}`^~``)
```
Comment (TheLostLambda) [inline concepts/characters/introduction.md:12]: `#\Newline` could use some code tags here :)

**Review by TheLostLambda — APPROVED** [review]
Overall LGTM! Some missing code tags if you want to fix that up too :)

File: exercises/concept/character-study/.meta/config.json (hunk 1 above)
Comment (verdammelt) [reply exercises/concept/character-study/.meta/config.json:8]: also i can't spell 'helping' it seems!

Comment (verdammelt) [reply exercises/concept/character-study/.meta/config.json]: Appears that at least for this track using a period may be more common so I'll keep the period in this case.

**Pull request merged**